stable/prometheus-operator/templates/prometheus/serviceperreplica.yaml

```diff
@@ -31,7 +32,7 @@ items:
       {{- end }}
       {{- end }}
       ports:
-        - name: {{ .Values.prometheus.prometheusSpec.portName }}
+        - name: {{ $portName }}
```

Comment:
There's actually a simpler way to fix this without introducing a variable: `$.Values.prometheus.prometheusSpec.portName`

According to the Helm [docs](https://helm.sh/docs/chart_template_guide/variables/):

> However, there is one variable that is always global - $ - this variable will always point to the root context. This can be very useful when you are looping in a range need to know the chart’s release name.

Reply:
Following your suggestion, I've changed the variable field to use the $ global scope context.